app/src/main/java/org/dosomething/letsdothis/tasks/DbGetUserTask.java

```diff
@@ -10,12 +10,12 @@
 /**
  * Created by toidiu on 4/17/15.
  */
-public class DbGetUser extends Task
+public class DbGetUserTask extends Task
```

Comment:
To discuss: if a previous task is taking a long time, does this task get unnecessarily delayed? Should we be using a loader instead?